RHDEVDOCS-3623: Update Pipeline section in add flow and pipeline docs (#42868)
openshift/openshift-docs: cicd/pipelines/working-with-pipelines-using-the-developer-perspective.adoc

include::modules/op-constructing-pipelines-using-pipeline-builder.adoc[leveloffset=+1]

== Creating OpenShift Pipelines along with applications

> To create pipelines along with applications, use the *From Git* option in the *Add+* view of the *Developer* perspective. You see a default pipeline selected and can click *Show Pipeline Visualization* to see the pipeline for the application. You can view all of your available pipelines and select the pipelines you want to use to create applications while importing your code or deploying an image. If an administrator has opted in for the Tekton Hub Integration, you can see tasks from Tekton Hub that are supported by your cluster. You can also check whether a webhook URL exists for a generated pipeline.

Reviewer: @karthikjeeyar can you please confirm if the last three sentences are technically correct?

> To create pipelines along with applications, use the *From Git* option in the *Add+* view of the *Developer* perspective. You see a default pipeline selected and can click *Show Pipeline Visualization* to see the pipeline for the application. You can view all of your available pipelines and select the pipelines you want to use to create applications while importing your code or deploying an image. If an administrator has opted in for the Tekton Hub Integration, you can see tasks from Tekton Hub that are supported by your cluster. You can also check whether a webhook URL exists for a generated pipeline.

Reviewer 2: We could also add a point about default webhooks that are created when the user goes through the add flow.
eg:
Default webhooks are added for the pipelines that are created using Add flow and URL is visible in the side panel of the selected resources in the Topology view.

> To create pipelines along with applications, use the *From Git* option in the *Add+* view of the *Developer* perspective. You can view all of your available pipelines and select the pipelines you want to use to create applications, while importing your code or deploying an image. You will also find a default pipeline selected and can click *Show Pipeline Visualization* to see the pipeline for the application. If an administrator has opted out of Tekton Hub integration, you will be able to see tasks from Tekton Hub which are supported by the cluster. You can also check if a webhook URL exists for a generated pipeline.

Reviewer 3: Question: On 2nd thought, I am not sure about the order of the events here that I get to see. What is the 1st thing that I get to see? This is my understanding: A default pipeline that is selected for the application? I understand that despite this default pipeline, I as a user am able to view all the available pipelines and select the one I want to create applications. Correct? 

If so, please rewrite it to suit this order. Something like this:

```suggestion
To create pipelines along with applications, use the *From Git* option in the *Add+* view of the *Developer* perspective. You see a default pipeline selected and can click *Show Pipeline Visualization* to see the pipeline for the application. You can view all of your available pipelines and select the pipelines you want to use to create applications while importing your code or deploying an image. If an administrator has opted out of Tekton Hub integration, you can see tasks from Tekton Hub which are supported by the cluster. You can also check whether a webhook URL exists for a generated pipeline.
```
